### PR TITLE
Add vint

### PR DESCRIPTION
**Pull request by liuchengxu**

### CODE REVIEW

**Comment by alerque** [comment]
By coincidence I just setup Vint as a CI job on another vim plugin yesterday, but using GitHub Actions. If you happen to be interested in that instead of Travis feel free to rip off the implementation [from here](https://github.com/majutsushi/tagbar/blob/master/.github/workflows/vint.yml). The end to end run time for the CI job is about 20s to fire up the VM, install Python and Vint, lint the plugin, and return.

**Comment by liuchengxu** [comment]
@alerque Thanks, I'm going to steal it then :(. And good to see you take the maintainance of tagbar :+1: .

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 sudo: required
```
Comment (alerque) [inline .travis.yml]: Maybe &lt;del&gt; this file too? You probably don't need both. Note that the GA one won't run on _this PR_, the workflow has to land on master before it considers running it on PRs.

Comment (liuchengxu) [reply .travis.yml]: Yeah, will drop that when everything is ready.

**Pull request merged**